src/modules/main/controllers/admin-write.controller.ts

```diff
@@ -94,8 +94,34 @@ export class AdminWriteController {
       exhibitions: mapEmpty(dto.exhibitions, (list) => list.map(id => ({ id })), []),
       image: mapEmpty(dto.image, image => ({ id: randomUUID(), buffer: image.buffer, mimeType: image.mimeType }), ArtworkImage.empty),
       protectedImage: (dto.image !== undefined) ? ArtworkImage.empty : undefined,
-    });
-    return { id: artwork.id };
+    };
+
+    const artworkDb = await this.adminRepository.saveArtwork(artwork);
+
+    let nft = null;

```

Comment:
please revert changes to this endpoint and create separate endpoint that will do this logic